### ISSUE TITLE
Sendtoaddress cli command generates a string, not clear what it is

### ISSUE THREAD

**Issue by adiack** [ElementsProject/elementsproject.github.io#32]
No biggie but when sending an asset to e.g Bob as part of tutorial : issuing-assets
A string is generated, would be nice to know what it is 
```
 e1-cli sendtoaddress $E2DEMOADD 10 "" "" false Prepaid-Credit-Mobile
**cc6408d0070bc87898fd4df2dc88f90a9660d901920391d7fdf74645f5b87f92**
```
 Same comment when you use generate command
```
e1-cli generate 1
[
  "**512356e0179a3f122ee2fb7626e89ac2130d78b3584985ab28da1ec0fad4fdac**"
]
```

**Comment by wintercooled** [comment]
Hi @adiack - that's the transaction id. That's shown earlier in the tutorial as it is saved to a TXID variable but isn't explicitly mentioned. As you say, that could be helpful so I will add a line to this page probably where sendtoaddress and generate are first used to explain:

https://elementsproject.org/elements-code-tutorial/basic-operations

Thanks.
Matt.

**Comment by wintercooled** [comment]
Addressed with PR #33
